package.json

```diff
@@ -55,7 +55,7 @@
     "@types/glob": "^5.0.32",
     "@types/is": "0.0.20",
     "@types/methods": "^1.1.0",
-    "@types/mocha": "^5.0.0",
+    "@types/mocha": "5.2.1",
```

Comment:
Currently need to keep this version fixed. 5.2.2 seems to have compilation issues.